### DISCUSSION TITLE
Not an issue but a question

### DISCUSSION THREAD

**Discussion by simon88** [danak6jq/RAK3172#16] [General]
Hi @danak6jq ,
First of all thanks for your project, I'm trying to play with it and RAK3172_SIP board, I'm trying to generate PWM without success, have you already try it ? I'm trying to USE TIM1 with PWM output on CH1 and CH2 but I've nothing.
If I look in low level sub ghz driver I found Ùtil_timer`maybe it's coming from this level...

**Comment by danak6jq** [comment]
Are you making progress on this? @simon88 ?
